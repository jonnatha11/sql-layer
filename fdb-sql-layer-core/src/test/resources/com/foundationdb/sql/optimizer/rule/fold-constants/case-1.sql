SELECT CASE WHEN LCASE(NULL) THEN 'null' WHEN name IS NOT NULL THEN name ELSE 'default' END FROM customers
