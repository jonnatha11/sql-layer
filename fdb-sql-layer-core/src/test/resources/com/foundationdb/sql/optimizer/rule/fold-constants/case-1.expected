SelectQuery@6f3b625b
  ResultSet@23b67d6a[_SQL_COL_1]
    Project@21e3cc77[customers.name]
      Select@7284aa02[]
        TableSource@58d51a54(customers)
